wp-content/themes/workingnyc/views/newsletter.twig

```diff
@@ -25,7 +25,7 @@
     <section class="site-max-width">
       <div class="py-5 tablet:py-6">
         <div data-js="newsletter-form">
-          <form action="https://nyc.us18.list-manage.com/subscribe/post?u={{ constant('MAILCHIMP_ACCOUNT') }}&amp;id={{ constant('MAILCHIMP_AUDIENCE_ID') }}" id="mc-embedded-subscribe-form" method="post" target="_blank">
+          <form action="https://api.sendgrid.com/v3/marketing/contacts" id="mc-embedded-subscribe-form" method="post" target="_blank">
```

Comment:
- [ ] Can you remove this SendGrid URL? The user will be added through the API endpoint you created right?